bosh-setup/scripts/setup_env

```diff
@@ -57,6 +57,27 @@ sed -i "s/CLOUD_FOUNDRY_PUBLIC_IP/cf-ip/g" settings
 cp settings $home_dir
 
 sed -i "s/REPLACE_WITH_BOSH_DIRECOT_IP/$bosh_director_ip/g" deploy_bosh.sh
+
+REPLACE_WITH_REGISTRY_HTTP_PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
```

Comment:
Ok will work on this after hours tonight. (blocked during work hours).



Kind Regards,

*Haydon Ryan | Advisory Solutions Architect | Pivotal Cloud Foundry
Services *

Mobile: 415-400-9377 | Email: hryan@pivotal.io

<http://www.pivotal.io>

www.pivotal.io


On Thu, May 5, 2016 at 2:04 AM, Bin Xia <notifications@github.com> wrote:

> In bosh-setup/scripts/setup_env
> <https://github.com/Azure/azure-quickstart-templates/pull/1913#discussion_r62151299>
> :
>
> > @@ -57,6 +57,27 @@ sed -i "s/CLOUD_FOUNDRY_PUBLIC_IP/cf-ip/g" settings
> >  cp settings $home_dir
> >
> >  sed -i "s/REPLACE_WITH_BOSH_DIRECOT_IP/$bosh_director_ip/g" deploy_bosh.sh
> > +
> > +REPLACE_WITH_REGISTRY_HTTP_PASSWORD=$(cat /dev/urandom | tr -dc 'a-zA-Z0-9' | fold -w 32 | head -n 1)
>
> @haydonryan <https://github.com/haydonryan>
> From my test, the script hangs in cat /dev/urandom | tr -dc 'a-zA-Z0-9' |
> fold -w 32 | head -n 1.
> Could you reproduce the hanging by deploying the template with your PR?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Azure/azure-quickstart-templates/pull/1913/files/53e894ad258ed407df3b0fb8bc111037a624983a#r62151299>
>